1. Faça um Programa que peça dois números e imprima o maior deles.

In [4]:
x,y = int(input("Digite um numero")),int(input("Digite outro numero"))
if (x>y):
    print(x)
else:
    print(y)

Digite um numero4
Digite outro numero5
5


2. Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [7]:
letras = ['a','e','i','o','u']
x= input("Digite uma letra")

if (x in letras):
  print("letra ",x, " é vogal")
else:
  print("letra ",x, " é cosoante")

Digite uma letraa
letra  a  é vogal


3. Faça um programa para a leitura de duas notas parciais de um aluno. O programa
deve calcular a média alcançada por aluno e apresentar:
A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
A mensagem "Reprovado", se a média for menor do que sete;
A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [8]:
x,y=int(input("digite a primeira nota")), int(input("digite a segunda nota"))
total =  (x + y) / 2
print("Nota ",total)
if (total > 7):
  print("Aprovado")
elif (total < 7):
  print("Reprovado")
else:
  print("Aprovado com Distinção")

digite a primeira nota4
digite a segunda nota5
Nota  4.5
Reprovado


4. Faça um Programa que leia três números e mostre-os em ordem decrescente.

In [12]:
x,y,w=int(input("digite a primeiro numer")), int(input("digite a segundo numero")), int(input("digite a terceiro numero"))

if (x > y and x > w):
  if(y > w):
    print("ordem: ", x,",", y,",", w)
  else:
     print("ordem: ", x,",", w,",", y)
elif (y > x and y > w):
  if (x > w):
    print("Ordem: ", y,",", x,",", w)
  else:
    print("Ordem: ",y,",", w,",", x)
else:
  if(w > y and w > x):
    print("Ordem: ",w,',',x,',',y)
  else:
    print("Ordem: ",w,",",y,",",x)


digite a primeiro numer3
digite a segundo numero7
digite a terceiro numero5
Ordem:  7 , 5 , 3


5. As Organizações Tabajara resolveram dar um aumento de salário aos seus
colaboradores e lhe contrataram para desenvolver o programa que calculará os
reajustes.
Faça um programa que recebe o salário de um colaborador e o reajuste segundo o
seguinte critério, baseado no salário atual:
salários até R$ 280,00 (incluindo) : aumento de 20%
salários entre R$ 280,00 e R$ 700,00 : aumento de 15%
salários entre R$ 700,00 e R$ 1500,00 : aumento de 10%
salários de R$ 1500,00 em diante : aumento de 5% Após o aumento ser realizado,
informe na tela:
o salário antes do reajuste;
o percentual de aumento aplicado;
o valor do aumento;
o novo salário, após o aumento.

In [13]:
salario = float(input("Digite o salario"))
percentualAumento = float(0)
aumento = float(0)
salarioAumento = float(0)

if(salario <= 280):
  percentualAumento = 20
  aumento = (salario / 100) * percentualAumento
  salarioAumento = aumento + salario
elif(salario >= 280 and salario < 700):
  percentualAumento = 15
  aumento = (salario / 100) * percentualAumento
  salarioAumento = aumento + salario
elif(salario >= 700 and salario < 1500):
  percentualAumento = 10
  aumento = (salario / 100) * percentualAumento
  salarioAumento = aumento + salario
elif(salario >= 1500):
  percentualAumento = 5
  aumento = (salario / 100) * percentualAumento
  salarioAumento = aumento + salario

print("o salário antes do reajuste ", salario)
print("o percentual de aumento aplicado", percentualAumento,"%")
print("o valor do aumento", aumento)
print("o novo salário, após o aumento", salarioAumento)

Digite o salario1200
o salário antes do reajuste  1200.0
o percentual de aumento aplicado 10 %
o valor do aumento 120.0
o novo salário, após o aumento 1320.0


6. Faça um Programa que leia um número e exiba o dia correspondente da semana.
(1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido.

In [14]:
dia = int(input("digite um numero"))

if(dia==1):
  print("Segunda")
elif(dia==2):
  print("Terça")
elif(dia==3):
  print("Quarta")
elif(dia==4):
  print("Quinta")
elif(dia==5):
  print("Sexta")
elif(dia==6):
  print("Sabado")
elif(dia==7):
  print("Domingo")

digite um numero5
Sexta


7. Faça um programa que lê as duas notas parciais obtidas por um aluno numa
disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos
obedece à tabela abaixo:
 Média de Aproveitamento Conceito
 Entre 9.0 e 10.0 A
 Entre 7.5 e 9.0 B
 Entre 6.0 e 7.5 C
 Entre 4.0 e 6.0 D
 Entre 4.0 e zero E
O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a
mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito
for D ou E.

In [15]:
x,y=int(input("digite a primeira nota")), int(input("digite a segunda nota"))
total =  (x + y) / 2
print("Nota ",total)
if (total >= 9 and total <= 10):
  print("A aprovado")
elif (total >= 7.5 and total < 9 ):
  print("B aprovado")
elif (total >= 6 and total < 7.5 ):
  print("C aprovado")
elif (total >= 4 and total < 6 ):
  print("D reprovado")
elif (total >= 0 and total < 4 ):
  print("E reprovado")

digite a primeira nota7
digite a segunda nota8
Nota  7.5
B aprovado


8. Faça um Programa que peça os 3 lados de um triângulo. O programa deverá
informar se os valores podem ser um triângulo. Indique, caso os lados formem um
triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:
Três lados formam um triângulo quando a soma de quaisquer dois lados for maior
que o terceiro;
Triângulo Equilátero: três lados iguais;
Triângulo Isósceles: quaisquer dois lados iguais;
Triângulo Escaleno: três lados diferentes;

In [16]:
lado1 = float(input("Digite um lado"))
lado2 = float(input("Digite segundo lado"))
lado3 = float(input("Digite terceiro lado"))

if(lado1 == lado2 and lado2 == lado3 and lado1 == lado3):
  print("Triângulo Equilátero")
elif(lado1 != lado2 and lado2 != lado3 and lado1 != lado3):
  print("Triângulo Isósceles")
else:
  print("Triângulo Escaleno")


Digite um lado4
Digite segundo lado4
Digite terceiro lado4
Triângulo Equilátero


9. Faça um programa que calcule as raízes de uma equação do segundo grau, na
forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as
consistências, informando ao usuário nas seguintes situações:
Se o usuário informar o valor de A igual a zero, a equação não é do segundo grau e
o programa não deve fazer pedir os demais valores, sendo encerrado;
Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao
usuário e encerre o programa;
Se o delta calculado for igual a zero a equação possui apenas uma raiz real;
informe-a ao usuário;
Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;

In [1]:
def delta(a,b,c):
    return (b**2) - (4*a*c)
def raizes(a,b,c,delta):
    x = (-b+ delta**(1.0/2.0)) / (2.0*a)
    y = (-b- delta**(1.0/2.0)) / (2.0*a)
    return {"x":x,"y":y}

a = int(input("Digitet numero"))

if (a == 0):
  print("programa encerrado")
else:
  b = int(input("Digitet numero"))
  c = int(input("Digitet numero"))
  if(delta(a,b,c) < 0):
    print("não tem raiz real")
  elif (delta(a,b,c) == 0):
    print("apenas uma raiz real")
  elif(delta(a,b,c > 0)):
    raiz = raizes(a,b,c,delta(a,b,c))
    print("a equal possui raiz real: ", raiz['x'], " e ", raiz['y'])


Digitet numero6
Digitet numero18
Digitet numero55
não tem raiz real


10. Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao
usuário a valor do saque e depois informar quantas notas de cada valor serão
fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é
de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a
quantidade de notas existentes na máquina.
Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de
100, uma nota de 50, uma nota de 5 e uma nota de 1;
Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de
100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.

In [2]:
notas = [1, 5, 10, 50, 100]
valor = int(input('Qual o valor? (validos entre 10 e 600)'))

if valor >= 10 and valor <= 600:
    quantia = 0
    saida = list()
    p = 4
    while quantia < valor and p >= 0:
        if valor - notas[p] >= 0:
            saida.append(notas[p])
            valor -= notas[p]
        else:
            p -= 1
    print('As notas são: ', saida)
else:
    print('Valor inválido.')

Qual o valor? (validos entre 10 e 600)500
As notas são:  [100, 100, 100, 100, 100]


11. Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As
perguntas são:
"Telefonou para a vítima?"
"Esteve no local do crime?"
"Mora perto da vítima?"
"Devia para a vítima?"
"Já trabalhou com a vítima?"
O programa deve no final emitir uma classificação sobre a participação da pessoa no
crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada
como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário,
ele será classificado como "Inocente".

In [3]:
interrogatorio = {"Telefonou para a vítima?":False,
    "Esteve no local do crime?":False, 
    "Mora perto da vítima?":False,
    "Devia para a vítima?":False,
    "Já trabalhou com a vítima?":False}
for item in interrogatorio:
    interrogatorio[item]=input(item + 'S/N :').upper()=='S'
respostas = (sum(interrogatorio.values()))

if respostas == 0 or respostas == 1:
    situacao = 'inocente'
elif respostas == 2:
    situacao = 'suspeito'
elif respostas == 3 or respostas == 4:
    situacao = 'cúmplice'
elif respostas == 5:
    situacao = 'assassino'
    
print(f'O investigado é {situacao}.')

Telefonou para a vítima?S/N :n
Esteve no local do crime?S/N :s
Mora perto da vítima?S/N :s
Devia para a vítima?S/N :s
Já trabalhou com a vítima?S/N :s
O investigado é cúmplice.


12. Uma fruteira está vendendo frutas com a seguinte tabela de preços:
 Até 5 Kg Acima de 5 Kg
Morango R$ 2,50 por Kg R$ 2,20 por Kg
Maçã R$ 1,80 por Kg R$ 1,50 por Kg
Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar
R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um
algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de
maças adquiridas e escreva o valor a ser pago pelo cliente

In [7]:
kgMorango = float(input("quantos irá comprar de morango"))
kgMaca = float(input("quantos irá comprar de maçã"))

total = 0

if (kgMorango <= 5):
  total += kgMorango * 2.50
if(kgMorango > 5):
  total += kgMorango * 2.20
if (kgMaca <= 5):
  total += kgMaca * 1.80
if(kgMaca > 5):
  total += kgMaca * 1.50

if ((kgMaca+kgMorango) >= 8 or total >= 25):
  desconto = total * 0.10

print("Morango: ",kgMorango,"kg")
print("Maçã: ",kgMaca,"kg")
print("Desconto: ",desconto," total ", (total - desconto))

quantos irá comprar de morango1
quantos irá comprar de maçã20
Morango:  1.0 kg
Maçã:  20.0 kg
Desconto:  3.25  total  29.25


13. Faça um programa que peça uma nota, entre zero e dez. Mostre uma mensagem
caso o valor seja inválido e continue pedindo até que o usuário informe um valor
válido.

In [8]:

while (int(input("Digite numero")) not in [0,1,2,3,4,5,6,7,8,9,10]):
  print("Valor invalido, tente novamente")
print("Nota aceita")

Digite numero44
Valor invalido, tente novamente
Digite numero5
Nota aceita


13. Faça um programa que leia e valide as seguintes informações:
Nome: maior que 3 caracteres;
Idade: entre 0 e 150;
Salário: maior que zero;
Sexo: 'f' ou 'm';
Estado Civil: 's', 'c', 'v', 'd';

In [9]:
def validar():
  nome = input("Digite se nome")
  idade = int(input("digite a idade"))
  salario = float(input("Digite o salario"))
  sexo = input("digite seu sexo f ou m")
  estado = input("Digite seu estado civil")

  req = {'nome':3,'idade':[0,150],'salario':0,'sexo':['f','m'],'estado':['s','c','v','d']}

  if(len(nome) <= req['nome']):
    print("Nome precisa ter mais que 3 letras")
    return False
  if(idade <= req['idade'][0] or idade > req['idade'][1]):
    print("Idade precisa estar entre 0 a 150")
    return False
  if(salario < req['salario']):
    print("Salario tem que ser maior que 0")
    return False
  if(sexo not in req['sexo']):
    print("digite f = female, m = male")
    return False
  if(estado not in req['estado']):
    print("digite s = solteiro, c = casado, v = viúvo, d = divorciado")
    return False
  return True

while(validar() != True):
  print("Tente novamente")
print("validado")

Digite se nomeju
digite a idade500
Digite o salario-20
digite seu sexo f ou mM
Digite seu estado civilS
Nome precisa ter mais que 3 letras
Tente novamente
Digite se nomejuliano
digite a idade32
Digite o salario20000
digite seu sexo f ou mm
Digite seu estado civils
validado


14. Faça um programa que leia 5 números e informe o maior número.

In [10]:
lista = []
maior = 0;

for x in range(5):
  lista.append(int(input("Digite um numero")))
  if(lista[x] > maior):
    maior = lista[x]

print(maior)

Digite um numero4
Digite um numero9
Digite um numero7
Digite um numero2
Digite um numero5
9


15. Faça um programa que leia 5 números e informe a soma e a média dos números.

In [11]:
soma = 0
for n in range(5):
    soma += int(input("Escreva um numero: "))
print("soma: ", soma)
print("media: ", soma/5)

Escreva um numero: 2
Escreva um numero: 3
Escreva um numero: 2
Escreva um numero: 3
Escreva um numero: 2
soma:  12
media:  2.4


16. Faça um programa que imprima na tela apenas os números ímpares entre 1 e 50.


In [13]:
lista=[]
for x in range(50): 
  if((x % 2) != 0):
    lista.append(x)
print(lista)

[1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47, 49]


17. Desenvolva um gerador de tabuada, capaz de gerar a tabuada de qualquer
número inteiro entre 1 a 10. O usuário deve informar de qual numero ele deseja ver
a tabuada. A saída deve ser conforme o exemplo abaixo:
Tabuada de 5:
5 X 1 = 5
5 X 2 = 10
...
5 X 10 = 50

In [14]:
numero = int(input("Digite um numero"))
for x in range(10):
  print(numero, " x ", (x+1), " = ", numero * (x+1))

Digite um numero7
7  x  1  =  7
7  x  2  =  14
7  x  3  =  21
7  x  4  =  28
7  x  5  =  35
7  x  6  =  42
7  x  7  =  49
7  x  8  =  56
7  x  9  =  63
7  x  10  =  70


18. A série de Fibonacci é formada pela seqüência 1,1,2,3,5,8,13,21,34,55,... Faça um
programa capaz de gerar a série até o n−ésimo termo.

In [15]:
n = 10
lista = [1,1]
for i in range(2,n):
    lista.append(lista[i-2]+lista[i-1])
if n == 1:
    print([1])
else:
    print(lista)

[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]


19. Faça um programa que calcule o fatorial de um número inteiro fornecido pelo
usuário. Ex.: 5!=5.4.3.2.1=120

In [16]:
n = int(input("Digite o valor de n: "))
fat = 1
i = 1
while i <= n:
  fat = fat*i
  print(n,"! ",i,"=", fat)
  i = i + 1
print("O valor de %d! eh =" %n, fat)

Digite o valor de n: 5
5 !  1 = 1
5 !  2 = 2
5 !  3 = 6
5 !  4 = 24
5 !  5 = 120
O valor de 5! eh = 120


20. O Sr. Manoel Joaquim possui uma grande loja de artigos de R$ 1,99, com cerca
de 10 caixas. Para agilizar o cálculo de quanto cada cliente deve pagar ele
desenvolveu um tabela que contém o número de itens que o cliente comprou e ao
lado o valor da conta. Desta forma a atendente do caixa precisa apenas contar
quantos itens o cliente está levando e olhar na tabela de preços. Você foi contratado
para desenvolver o programa que monta esta tabela de preços, que conterá os
preços de 1 até 50 produtos, conforme o exemplo abaixo:
Lojas Quase Dois - Tabela de preços
1 - R$ 1.99
2 - R$ 3.98
...
50 - R$ 99.50

In [17]:
lista=[]
for x in range(50): 
  print((x+1),"- R$ ",((x+1)*1.99))

1 - R$  1.99
2 - R$  3.98
3 - R$  5.97
4 - R$  7.96
5 - R$  9.95
6 - R$  11.94
7 - R$  13.93
8 - R$  15.92
9 - R$  17.91
10 - R$  19.9
11 - R$  21.89
12 - R$  23.88
13 - R$  25.87
14 - R$  27.86
15 - R$  29.85
16 - R$  31.84
17 - R$  33.83
18 - R$  35.82
19 - R$  37.81
20 - R$  39.8
21 - R$  41.79
22 - R$  43.78
23 - R$  45.77
24 - R$  47.76
25 - R$  49.75
26 - R$  51.74
27 - R$  53.73
28 - R$  55.72
29 - R$  57.71
30 - R$  59.7
31 - R$  61.69
32 - R$  63.68
33 - R$  65.67
34 - R$  67.66
35 - R$  69.65
36 - R$  71.64
37 - R$  73.63
38 - R$  75.62
39 - R$  77.61
40 - R$  79.6
41 - R$  81.59
42 - R$  83.58
43 - R$  85.57
44 - R$  87.56
45 - R$  89.55
46 - R$  91.54
47 - R$  93.53
48 - R$  95.52
49 - R$  97.51
50 - R$  99.5


21. O Sr. Manoel Joaquim acaba de adquirir uma panificadora e pretende implantar
a metodologia da tabelinha, que já é um sucesso na sua loja de 1,99. Você foi
contratado para desenvolver o programa que monta a tabela de preços de pães, de
1 até 50 pães, a partir do preço do pão informado pelo usuário, conforme o exemplo
abaixo:
Preço do pão: R$ 0.18
Panificadora Pão de Ontem - Tabela de preços
1 - R$ 0.18
2 - R$ 0.36
...
50 - R$ 9.00

In [21]:
lista=[]
for x in range(50): 
  print((x+1),"- R$ ","%.2f"%((x+1)*0.18))

1 - R$  0.18
2 - R$  0.36
3 - R$  0.54
4 - R$  0.72
5 - R$  0.90
6 - R$  1.08
7 - R$  1.26
8 - R$  1.44
9 - R$  1.62
10 - R$  1.80
11 - R$  1.98
12 - R$  2.16
13 - R$  2.34
14 - R$  2.52
15 - R$  2.70
16 - R$  2.88
17 - R$  3.06
18 - R$  3.24
19 - R$  3.42
20 - R$  3.60
21 - R$  3.78
22 - R$  3.96
23 - R$  4.14
24 - R$  4.32
25 - R$  4.50
26 - R$  4.68
27 - R$  4.86
28 - R$  5.04
29 - R$  5.22
30 - R$  5.40
31 - R$  5.58
32 - R$  5.76
33 - R$  5.94
34 - R$  6.12
35 - R$  6.30
36 - R$  6.48
37 - R$  6.66
38 - R$  6.84
39 - R$  7.02
40 - R$  7.20
41 - R$  7.38
42 - R$  7.56
43 - R$  7.74
44 - R$  7.92
45 - R$  8.10
46 - R$  8.28
47 - R$  8.46
48 - R$  8.64
49 - R$  8.82
50 - R$  9.00


22. O Sr. Manoel Joaquim expandiu seus negócios para além dos negócios de 1,99 e
agora possui uma loja de conveniências. Faça um programa que implemente uma
caixa registradora rudimentar. O programa deverá receber um número
desconhecido de valores referentes aos preços das mercadorias. Um valor zero deve
ser informado pelo operador para indicar o final da compra. O programa deve então
mostrar o total da compra e perguntar o valor em dinheiro que o cliente forneceu,
para então calcular e mostrar o valor do troco. Após esta operação, o programa
deverá voltar ao ponto inicial, para registrar a próxima compra. A saída deve ser
conforme o exemplo abaixo:
Lojas Tabajara
Produto 1: R$ 2.20
Produto 2: R$ 5.80
Produto 3: R$ 0
Total: R$ 9.00
Dinheiro: R$ 20.00
Troco: R$ 11.00
...

In [22]:
def adicionar(total):
  num = int(input("Digite valor"))
  if(num > 0):
    total.append(num)
    return True
  else:
    return False
  
tot=[]
dinheiro = 0
valor =0

while(adicionar(tot)):
  print("Adicionado")
dinheiro = float(input("Digite o valor pago"))

print("lojas tabajaras")
for x in range(len(tot)):
  valor = valor + tot[x]
  print("produto ",(x+1),": R$",tot[x])

print("total: ", valor)
print("dinheiro pago: ", dinheiro)
print("troco: ", (dinheiro - valor))

Digite valor4
Adicionado
Digite valor5
Adicionado
Digite valor-1
Digite o valor pago20
lojas tabajaras
produto  1 : R$ 4
produto  2 : R$ 5
total:  9
dinheiro pago:  20.0
total:  11.0


23. Foi feita uma estatística em cinco cidades brasileiras para coletar dados sobre
acidentes de trânsito. Foram obtidos os seguintes dados:
Código da cidade;
Número de veículos de passeio (em 1999);
Número de acidentes de trânsito com vítimas (em 1999). Deseja-se saber:
Qual o maior e menor índice de acidentes de transito e a que cidade pertence;
Qual a média de veículos nas cinco cidades juntas;
Qual a média de acidentes de trânsito nas cidades com menos de 2.000 veículos de
passeio.

In [23]:
cidade1 = {'codigo':1,'carros':1000,'acidentes':78}
cidade2 = {'codigo':2,'carros':1600,'acidentes':230}
cidade3 = {'codigo':3,'carros':1400,'acidentes':111}
cidade4 = {'codigo':4,'carros':500,'acidentes':10}
cidade5 = {'codigo':5,'carros':2100,'acidentes':295}

cidades = [cidade1,cidade2,cidade3,cidade4,cidade5]

maior_indice = {'id':0,'acidentes':0}
menorindice = {'id':0,'acidentes':999999999}
media_veiculos = 0
media_acidentes_menos_2000 = {'cont':0,'acidentes':0}

for x in cidades:
  media_veiculos = media_veiculos + x['carros']
  if (x['acidentes'] < menorindice['acidentes']):
    menorindice['acidentes'] = x['acidentes']
    menorindice['id'] = x['codigo']
  if (x['acidentes'] > maior_indice['acidentes']):
    maior_indice['acidentes'] = x['acidentes']
    maior_indice['id'] = x['codigo']
  if(x['carros'] < 2000):
    media_acidentes_menos_2000['cont'] += 1
    media_acidentes_menos_2000['acidentes'] += x['acidentes']

print("menor indice", menorindice['acidentes'],'cidade', menorindice['id'])
print("maior indice", maior_indice['acidentes'],'cidade', maior_indice['id'])
print('Média de veiculos de passeio: ', media_veiculos / 5)
print('media de acidentes em cidades com menos de 2000 carros de passeio',media_acidentes_menos_2000['acidentes'] / media_acidentes_menos_2000['cont'])

menor indice 10 cidade 4
maior indice 295 cidade 5
Média de veiculos de passeio:  1320.0
media de acidentes em cidades com menos de 2000 carros de passeio 107.25


24. Faça um programa que receba o valor de uma dívida e mostre uma tabela com
os seguintes dados: valor da dívida, valor dos juros, quantidade de parcelas e valor
da parcela.
Os juros e a quantidade de parcelas seguem a tabela abaixo:
Quantidade de Parcelas % de Juros sobre o valor inicial da dívida
1 0
3 10
6 15
9 20
12 25
Exemplo de saída do programa:
Valor da Dívida Valor dos Juros Quantidade de Parcelas Valor da Parcela
R$ 1.000,00 0 1 R$ 1.000,00
R$ 1.100,00 100 3 R$ 366,00
R$ 1.150,00 150 6 R$ 191,67

In [25]:
divida = float(input("Digite valor da divida"))
juros = 0
parcelas = 3
for x in range(5):
  print('Valor da Divida',divida,
  'valor do juros ', juros*(divida/100),
  'quantidade de apercelas', parcelas * (x),
  ' valor da parcela ', divida+(juros*(divida/100))
  )
  juros +=5

Digite valor da divida50
Valor da Divida 50.0 valor do juros  0.0 quantidade de apercelas 0  valor da parcela  50.0
Valor da Divida 50.0 valor do juros  2.5 quantidade de apercelas 3  valor da parcela  52.5
Valor da Divida 50.0 valor do juros  5.0 quantidade de apercelas 6  valor da parcela  55.0
Valor da Divida 50.0 valor do juros  7.5 quantidade de apercelas 9  valor da parcela  57.5
Valor da Divida 50.0 valor do juros  10.0 quantidade de apercelas 12  valor da parcela  60.0


25. Faça um programa que mostre os n termos da Série a seguir:
 S = 1/1 + 2/3 + 3/5 + 4/7 + 5/9 + ... + n/m.
Imprima no final a soma da série.

In [26]:
x=1
y=1
n=0
total=0
for i in range(5):
  n = x/y
  print(x,'/',y)
  x+=1
  y+=2
  total += n
print('total: %.2f' % total)

1 / 1
2 / 3
3 / 5
4 / 7
5 / 9
total: 3.39
